src/events/events.ts

```diff
@@ -1,9 +1,7 @@
 import { WorkOS } from '../workos';
-import { Event, EventResponse } from './interfaces';
 import { ListEventOptions } from './interfaces';
-import { deserializeList } from '../common/serializers';
-import { List, ListResponse } from '../common/interfaces';
-import { deserializeEvent } from './serializers/event.serializer';
+import { deserializeEvent, deserializeList } from '../common/serializers';
```

Comment:
This change is important. Just by swapping the import location of the `deserializeEvent` we're now swapping out for the `deserializeEvent` that I wrote for webhooks. This fixes https://linear.app/workos/issue/USRLD-589/deserialize-event-responses